R/zzz.R

```diff
@@ -1,4 +1,4 @@
-globalVariables(c(".", ".data"))
+globalVariables(c(".", ".data", ".rename_col", ".original_col","df_names","new_name_in_df"))
```

Comment:
@thebioengineer Can you explain to me why these are necessary to put here?

Reply:
R CMD CHECK was complaining about no global reference to these. It is due to the tidyverse code I am using in select_col_plan.